mage_ai/data_preparation/models/pipelines/integration_pipeline.py

```diff
@@ -111,6 +111,8 @@ def source_file_path(self) -> str:
     @property
     def settings_file_path(self) -> str:
         if self.data_integration and 'catalog' in self.data_integration:
+            if os.path.exists(self.catalog_config_path):
```

Comment:
simply changing the file path could still cause problem since the file format also changes. we'll need to update mage_integration module to read from json file instead. https://github.com/mage-ai/mage-ai/blob/master/mage_integrations/mage_integrations/sources/utils.py#L279-L281